include/seqan3/range/views/pairwise_combine.hpp

```diff
@@ -219,7 +219,8 @@ class pairwise_combine_view : public std::ranges::view_interface<pairwise_combin
         requires std::ranges::sized_range<underlying_range_type>
     //!\endcond
     {
-        return (std::ranges::size(u_range) * (std::ranges::size(u_range) - 1) / 2);
+        auto const size = std::ranges::size(u_range);
```

Comment:
Since `size()` is constant. Does this PR really improve the performance? (Just asking, I don't care for either version evenif it does not)

Reply:
Nop; it shouldn't improve performance since it is constant time. It's more like a style guide, [FIX] might be the wrong tag for this.

Reply:
Alright.